v2.0/app/containers/App/fire-config.js

```diff
@@ -2,7 +2,8 @@ import firebase from 'firebase/app';
 import 'firebase/auth';
 import { FIREBASE_CONFIG } from '../../config/constants';
 
-firebase.initializeApp(FIREBASE_CONFIG);
+//firebase.initializeApp(FIREBASE_CONFIG);
+!firebase.apps.length ? firebase.initializeApp(FIREBASE_CONFIG) : firebase.app()
```

Comment:
why are we doing this?

Reply:
also there are some merge conflicts?

Reply:
I will resolve the merge conflicts.
The change regarding firebase.apps.length is to prevent firebase getting initialized multiple times.  It fixed a console error I observed.